app/components/DocumentListItem.js

```diff
@@ -63,6 +64,10 @@ function DocumentListItem(props: Props, ref) {
   const queryIsInTitle =
     !!highlight &&
     !!document.title.toLowerCase().includes(highlight.toLowerCase());
+  const queryIsInContext =
+    !!highlight && !!context?.toLowerCase().includes(highlight?.toLowerCase());
+  const showPreviousTitleTip =
+    highlight && !queryIsInTitle && !queryIsInContext;
```

Comment:
Works without changing the API. What do you think about this?

Reply:
I think it is pragmatic but deserving of a code comment as it's a loose coupling that may become an unreliable assumption in the future. For this PR it makes sense!